web/src/app/projects/monitor-summary/monitor-summary.component.ts

```diff
@@ -12,27 +12,23 @@ import { MonitorModel } from '../../shared/models/index.model';
   templateUrl: './monitor-summary.component.html',
   styleUrls: ['./monitor-summary.component.scss'],
 })
-export class MonitorSummaryComponent implements OnChanges {
+export class MonitorSummaryComponent  {
 
-  @Input()
-  public monitors: MonitorModel[] = [];
+  public _monitors: MonitorModel[] = [];
```

Comment:
I think `_` means it is private not public

Reply:
As the change detection was not working for Input() properties.
So I used setter here.
I have also fixed it using ngOnChanges() in previous commit.

Please check and let me know your feedback

Reply:
I think the `ngOnChanges` is the better option here. Let me have a read/play 🤓